db/schema.rb

```diff
@@ -155,6 +155,7 @@
     t.datetime "updated_at", null: false
     t.index ["account_id", "status_id"], name: "index_mentions_on_account_id_and_status_id", unique: true
     t.index ["status_id"], name: "index_mentions_on_status_id"
+    t.index ["status_id"], name: "mentions_status_id_index"
```

Comment:
@akihikodaki, I don't remember doing any schema change. Not sure how it got it. I'll remove this.